components/create/editor/Editor.tsx

```diff
@@ -101,528 +100,545 @@ export const MarketEditor = () => {
   const showLiquidityWarning =
     fieldsState.liquidity.isTouched && form.liquidity?.deploy && isWizard;
 
+  const isLoaded = Boolean(chainTime && isFetched);
+
   return (
-    <Transition
-      show={Boolean(chainTime && isFetched)}
-      enter="transition-opacity duration-100"
-      enterFrom="opacity-0"
-      enterTo="opacity-100"
-      leave="transition-opacity duration-100"
-      leaveFrom="opacity-100"
-      leaveTo="opacity-0"
-    >
-      <h2
-        className="relative font-3xl text-center flex justify-center items-center gap-3 mb-6"
-        ref={headerRef}
-      >
-        <div className="relative md:flex justify-center items-center">
-          Create Market
-          <EditorResetButton editor={editor} />
-        </div>
-      </h2>
-
-      <div className="flex center mb-8">
-        <div className="mr-3 font-light">One Page</div>
-        <Toggle checked={isWizard} onChange={toggleWizard} />
-        <div className="ml-3 font-light">Wizard</div>
-      </div>
-
-      <div className="mb-8 md:mb-12">
-        {isWizard && (
-          <WizardStepper
-            steps={steps}
-            current={currentStep}
-            onChange={(step) => setStep(step)}
+    <>
+      {isLoaded === false && (
```

Comment:
Diff is pretty horrible, I've just wrapped everything in `<></>` and added a loader here